src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -57,7 +57,9 @@
   </PropertyGroup>
   <!-- ********************** Configuration and Platform specific properties ********************* -->
   <!-- All Available Configurations-->
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
```

Comment:
Does it have to be Debug Configuration only?

Reply:
The "DocumentationFile "tag is auto generated by activating XML documentation option in VS. The xml file is generated when the project is build before calling any MSBuild command, but it could be easily added to "Release Configuration" as well manually.

Reply:
Let's add it as a part of build process, even for release configuration, so we can ensure latest xml is added to Nuget everytime.

I'm also interested in adding tests for xml validations, we'll likely take care of them in next PRs (post .Net Core documentation is complete).